lib/Models/Catalog/Gtfs/GtfsCatalogItem.ts

```diff
@@ -279,19 +306,44 @@ export default class GtfsCatalogItem extends MappableMixin(
       uri: new ConstantProperty(this.model.url),
       upAxis: new ConstantProperty(this._cesiumUpAxis),
       forwardAxis: new ConstantProperty(this._cesiumForwardAxis),
-      scale: new ConstantProperty(
-        this.model.scale !== undefined ? this.model.scale : 1
-      ),
+      scale: new ConstantProperty(this.model.scale ?? 1),
       heightReference: new ConstantProperty(HeightReference.RELATIVE_TO_GROUND),
       distanceDisplayCondition: new ConstantProperty({
         near: 0.0,
         far: this.model.maximumDistance
-      })
+      }),
+      maximumScale: new ConstantProperty(this.model.maximumScale),
+      minimumPixelSize: new ConstantProperty(this.model.minimumPixelSize ?? 0),
+      shadows: ShadowMode.DISABLED
     };
 
     return new ModelGraphics(options);
   }
 
+  @computed
+  private get _coloredModels() {
+    const colorGroups = this.model?.colorModelsByProperty?.colorGroups;
+    const model = this._model;
+    if (
+      !isDefined(model) ||
+      !isDefined(this.model?.colorModelsByProperty?.property) ||
+      !isDefined(colorGroups) ||
+      colorGroups.length === 0
+    ) {
+      return undefined;
+    }
+    return colorGroups.map(({ color }) => {
+      const coloredModel = model.clone();
+      coloredModel.color = new ConstantProperty(
+        Color.fromCssColorString(color ?? "white")
+      );
+      coloredModel.colorBlendMode = new ConstantProperty(
+        ColorBlendMode.REPLACE
+      );
```

Comment:
Just a suggestion: We could use `ColorBlendMode.MIX` and change the gltf material to something like:

```
"pbrMetallicRoughness": {
        "baseColorFactor": [
          1.0,
          1.0,
          1.0,
          0.5
        ],
        "metallicFactor": 1.0,
        "roughnessFactor": 0
      }
```

which in my opinion looks more "3d" :) 

![image](https://user-images.githubusercontent.com/1430/162353236-2bf8d6db-2fd6-495f-93fa-75ee080b9bb3.png)

Reply:
How do I change the gltf material?

Reply:
You can use https://github.com/CesiumGS/gltf-pipeline to convert the glb to gltf, edit and then back to glb.

Reply:
This does look good. I've updated my model and GtfsCatalogItem